code/modules/clothing/suits/jobs.dm

```diff
@@ -76,6 +76,27 @@
 	allowed = list(/obj/item/kitchen)
 	togglename = "sleeves"
 
+/obj/item/clothing/suit/toggle/chef/equipped(mob/user, slot)
+	. = ..()
+	if(!ishuman(user))
+		return
+	if(slot == slot_wear_suit)
+		var/mob/living/carbon/human/H = user
+		if (istype(H.get_item_by_slot(slot_head), /obj/item/clothing/head/chefhat))

```

Comment:
Does this include the cooks cap?

Reply:
If you mean the white cap, no. The white cap is a generic item and not tied to the job.

Reply:
also, 
![image](https://user-images.githubusercontent.com/20017308/36079124-597d6c9a-0f88-11e8-90a2-69e00ce180a3.png) this is not a cap

Reply:
There also isn't a jobcheck here, unlike for the hat equip.